test/views/pages/home/content/ranking/static_ranking_list_test.dart

```diff
@@ -0,0 +1,74 @@
+import "package:flutter/material.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/views/components/content/publication_header/user_profile_pop_up.dart";
+import "package:proxima/views/pages/home/content/ranking/ranking_card.dart";
+import "package:proxima/views/pages/home/content/ranking/ranking_list.dart";
+
+import "../../../../../mocks/data/ranking_data.dart";
+
+void main() {
+  testWidgets(
+    "Check that all the ranking elements are displayed correctly",
```

Comment:
I think it would be best to put this test in a group "Widgets display" and name the test with format "Display <elements>" like "Display ranking elements" so that it stays consistent with the rest of UI tests :)

Reply:
You are right 💯 

Fixed in: [9f652cc](https://github.com/ProximaEPFL/proxima/pull/322/commits/9f652cc91255a473b8126c34b7d1b479086c7c7a)

Thank you !